custom_components/eloverblik/const.py

```diff
@@ -1,3 +1,4 @@
 """Constants for the Eloverblik integration."""
 
 DOMAIN = "eloverblik"
+CURRENCY_KRONER_PER_KILO_WATT_HOUR = "kr/kWh"
```

Comment:
Would it not be more correct to use DKK/kWh? Also for the other places where kr/kWh is used.

Reply:
Hm in the HA Core constants they use the shorthand for currencies: https://github.com/home-assistant/core/blob/58d531841bc2197616e7285d0561a9935d9bda73/homeassistant/const.py#L504

E.g. € and $ vs EUR or USD, so I think "kr" is more in line with that?

Reply:
Let stick to kr/kWh for now.